examples/annotation_import/tiled.ipynb

```diff
@@ -38,34 +38,52 @@
       },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #22.    
</pre><p>I don't think this is necessary any more. I think it uploads in-line. Please double check if this is needed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/817/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
Removed !